server/graphql/v2/query/collection/AccountsQuery.ts

```diff
@@ -52,8 +53,9 @@ const AccountsQuery = {
       description: 'Limit the search to collectives belonging to these countries',
     },
     orderBy: {
-      type: OrderByInput,
+      type: new GraphQLNonNull(OrderByInput),
       description: 'The order of results',
+      defaultValue: { field: 'ACTIVITY', direction: 'DESC' },
```

Comment:
This would change the default sort field from `RANK` to `ACTIVITY`. We don't want to change the default behavior.